File: python/src/commands/info.py
import os
import platform
import sys

import click


@click.command()
def info():
    """Show system info."""
    data = {
        "platform": platform.system().lower(),
        "python": sys.version.split()[0],
        "arch": platform.machine(),
        "pid": str(os.getpid()),
    }
    for key, value in data.items():
        click.echo(f"  {click.style(key.ljust(10), fg='cyan')}  {value}")

File: go/cmd/info.go
package cmd

import (
	"fmt"
	"os"
	"runtime"

	"github.com/spf13/cobra"
)

var infoCmd = &cobra.Command{
	Use:   "info",
	Short: "Show system info",
	Run: func(cmd *cobra.Command, args []string) {
		pid := os.Getpid()
		info := []struct{ key, value string }{
			{"platform", runtime.GOOS},
			{"arch", runtime.GOARCH},
			{"go", runtime.Version()},
			{"pid", fmt.Sprintf("%d", pid)},
		}
		for _, item := range info {
			fmt.Printf("  %-10s  %s\n", item.key, item.value)
		}
	},
}
